### ISSUE TITLE
How to make demucs start by default

### ISSUE THREAD

**Issue by Andrew-afk** [facebookresearch/demucs#78]
Hi, i just wanna know how to instead of typing "conda activate demucs"
and moving to the audio folder
d:
cd Program
cd demucs

Make a bat file or something to start the anaconda prompt with demucs activated and the demucs folder opened. Have a good day!

**Comment by adefossez** [comment]
Yeah that's a great idea, but I don't know how to write this bat file and I don't have windows. If someone knows how to do that I would be happy to include it in the repo.
